src/components/FriendPage/FriendPage.js

```diff
@@ -34,8 +41,11 @@ const FriendPage = ({ id }) => {
 
   if (friend && active) {
     return (
-      <main className='friend-gift-page'>
+      <React.Fragment>
+      {!friendExist && <Error error={`This page does not exist. Click button to go back to main page.`} />}
+      <main className={friendExist ? 'friend-gift-page' : 'hidden'}>
         <header>
+          <Link to={'/'}><button>Go back to home page</button></Link>
```

Comment:
Thanks for adding this button! Maybe we can trim it down a little when styling so that it just says "Main" or "Dashboard", or potentially is just a back arrow.